antora.yml

```diff
@@ -1,6 +1,6 @@
 name: server
 title: ownCloud Server
-version: master
+version: '10.0'
```

Comment:
Sorry, I have pushed that fix directly to the branch by accident, please drop this change.

Reply:
I thought 10.1 is the one that you pushed.

Reply:
i had pushed both before i checked that i forgot to create a separate branch ;)

Reply:
np. thanks.

Reply:
fixed.